spec/features/feedback_form_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Feedback form', type: :feature, js: true do
+  before do
+    visit root_path
+  end
+
+  it 'is initially collapsed' do
+    expect(page).not_to have_css '#feedback-form.in'
+  end
+
+  it 'can be expanded' do
+    click_link 'Feedback', match: :first
+    expect(page).to have_css '#feedback-form.in'
+  end
+
+  it 'can be collapsed' do
+    click_link 'Feedback', match: :first
+    find('button.cancel-link').click
+    expect(page).not_to have_css '#feedback-form.in'
+  end
+
+  context 'when not logged in' do
+    it 'shows the reCAPTCHA challenge' do
+      click_link 'Feedback', match: :first
+      expect(page).to have_css '.library-hours-captcha'
+    end
+  end
+
+  context 'when logged in' do
+    before do
+      stub_current_user(build(:superadmin_user))
+    end
+
+    it 'does not show the reCAPTCHA challenge' do
+      click_link 'Feedback', match: :first
+      expect(page).not_to have_css '.library-hours-captcha'
```

Comment:
You need `visible: false` here, otherwise the test always passes. As written, I think the test checks that the element is NOT present AND visible. So it passes when you are not logged in and the element is present AND invisible. It also passes when the element is NOT present. This is masking the issue with the page load/log in order above.